evm-types.md

```diff
@@ -138,6 +138,9 @@ Primitives provide the basic conversion from K's sorts `Int` and `Bool` to EVM's
     rule bool2Word( B:Bool ) => 1 requires B
     rule bool2Word( B:Bool ) => 0 requires notBool B
 
+    rule bool2Word( B:Bool ) ==Int 1 => B         [simplification]
+    rule bool2Word( B:Bool ) ==Int 0 => notBool B [simplification]
+
```

Comment:
My guess is that these need to go in `tests/specs/lemmas.k`, in the appropriate module where only the Haskell backend will see them @ttuegel .

Reply:
Done!